src/libcore/str/mod.rs

```diff
@@ -2085,8 +2085,8 @@ impl str {
     /// let len = "foo".len();
     /// assert_eq!(3, len);
     ///
-    /// let len = "ƒoo".len(); // fancy f!
-    /// assert_eq!(4, len);
+    /// assert_eq!("ƒoo".len(), 4); // fancy f!
```

Comment:
is the rust convention to do `assert_eq!(expected, actual)`? or `assert_eq!(actual, expected)`?

it looks like String is doing one but str is doing the other